src/util/types.ts

```diff
@@ -31,10 +35,10 @@ export type StrengthMeterStyles = {
   100: { style: string; text: string };
 };
 
-export type MovieType = {
-  id: number;
+export type MovieProps = {
```

Comment:
The `Props` suffix is most often used to denote a type that is used to describe the property types on a react component. That's not the purpose of this type, right? I would have just called it `Movie`, unless you want to use some sort of consistent suffix on all of your data object types in which case I would maybe call it `MovieData`.